### DISCUSSION TITLE
Qiita で提供される HTML タグの一覧化

### DISCUSSION THREAD

**Discussion by yarakos95** [increments/qiita-discussions#334] [要望・アイデア・不具合]
Qiita で利用可能な HTML タグは複数ありますが、これを一覧にすることは可能でしょうか。

Markdown にとどまらず、HTML タグを利用したい・利用できるか検討するときに、このような HTML タグの一覧があると助かる場合があります。

実際、`<caption>` タグは 9 月 20 日に提供されていますが、リリースノートを見て把握しているユーザーはそう多くないと思います。新たに参入するユーザーであっても、過去のリリースノートをさかのぼって読む人は少ないでしょう。また、チートシートにも記載はありますが、慣れてくればチートシートを見ることもありません。

### `<kbd>` タグと `<samp>` タグ

Qiita で利用できる HTML タグを実際に確認しているときに気付きました。

`<kbd>` タグを利用すると等幅フォントになりますが、下の例のような影付き枠は表示されません。これは意図的でしょうか？

`<kbd>KBD</kbd>`: <kbd>KBD</kbd>

`<samp>` タグを利用することも出来ますが、これと見分けを付けることはないため、利用価値が下がっていると思います。

![image](https://user-images.githubusercontent.com/75878492/196723948-f61c2e3b-3a8b-4460-b584-41bb360204fa.png)

<details>
<summary>サンプルコード</summary>

```markdown
KBD: <kbd>KBD</kbd>
SAMP: <samp>SAMP</samp>
```

</details>

**Comment by tanaka-asami415** [comment]
@Yarakashi-Kikohshi 
ご意見ありがとうございます🙏

恐れ入りますが、タグについては数も多く内容も変わってしまうため、現時点では一覧化する予定はありません🙇
お問い合わせがよくある使用頻度の高そうなタグはチートシートに記載するようにしています。
更新の際に通知が届きますので、可能でしたらチートシートをストックしておいていただけると幸いです。

`<kbd>`タグについては、確認したところサービス開始時から入っているようで、
正直にお答えすると歴史的背景が分からず、意図的かどうかが分かりませんでした…🙏
おそらく問題ないと思いますので、スタイルがあたるように修正します！

**Reply by yarakos95** [reply]
予定がない件、了解しました。チートシート等確認しながら個人的に確認作業をしてみます 💪 

`<kbd>` タグについても了解しました。スタイルの修正よろしくお願いします 😄 

Markdown のみを利用することを思えば HTML タグのスタイルは大きく気にする必要はないとは言え、歴史的背景が分からない点には少し驚きました。タグが利用可能になっていて、スタイルもフォントが変更されるため、何かしらの意図があるものだと思っていました :thinking:
